win.patch

```diff
@@ -0,0 +1,338 @@
+From a7742455b2341b0603a431eca0200827eacb51d6 Mon Sep 17 00:00:00 2001
+From: XingShuo <928160430@qq.com>
+Date: Fri, 28 Jul 2023 14:23:16 +0800
+Subject: [PATCH] patch1
+
+---
+ worldgen/terrain/assets/landtiles/core.py                 | 2 +-
+ worldgen/terrain/source/cpu/elements/atmosphere.cpp       | 2 +-
+ worldgen/terrain/source/cpu/elements/ground.cpp           | 2 +-
+ worldgen/terrain/source/cpu/elements/landtiles.cpp        | 2 +-
+ worldgen/terrain/source/cpu/elements/mountains.cpp        | 2 +-
+ .../terrain/source/cpu/elements/upsidedown_mountains.cpp  | 2 +-
+ worldgen/terrain/source/cpu/elements/voronoi_rocks.cpp    | 2 +-
+ worldgen/terrain/source/cpu/elements/warped_rocks.cpp     | 2 +-
+ worldgen/terrain/source/cpu/elements/waterbody.cpp        | 2 +-
+ worldgen/terrain/source/cpu/meshing/visibility_test.cpp   | 2 +-
+ worldgen/terrain/source/cpu/surfaces/chunkyrock.cpp       | 2 +-
+ worldgen/terrain/source/cpu/surfaces/cobble_stone.cpp     | 2 +-
+ worldgen/terrain/source/cpu/surfaces/cracked_ground.cpp   | 2 +-
+ worldgen/terrain/source/cpu/surfaces/dirt.cpp             | 2 +-
+ worldgen/terrain/source/cpu/surfaces/ice.cpp              | 2 +-
+ worldgen/terrain/source/cpu/surfaces/mountain.cpp         | 2 +-
+ worldgen/terrain/source/cpu/surfaces/mud.cpp              | 2 +-
+ worldgen/terrain/source/cpu/surfaces/sand.cpp             | 2 +-
+ worldgen/terrain/source/cpu/surfaces/sandstone.cpp        | 2 +-
+ worldgen/terrain/source/cpu/surfaces/snow.cpp             | 2 +-
+ worldgen/terrain/source/cpu/surfaces/soil.cpp             | 2 +-
+ worldgen/terrain/source/cpu/surfaces/stone.cpp            | 2 +-
+ worldgen/terrain/utils/ctype_util.py                      | 8 +++++++-
+ 23 files changed, 29 insertions(+), 23 deletions(-)
+
+diff --git a/worldgen/terrain/assets/landtiles/core.py b/worldgen/terrain/assets/landtiles/core.py
+index aa2d075..5360b30 100644
+--- a/worldgen/terrain/assets/landtiles/core.py
++++ b/worldgen/terrain/assets/landtiles/core.py
+@@ -83,7 +83,7 @@ def assets_to_data(
+     N=2048,
+     do_smooth=False,
+ ):
+-    preset_name = str(folder).split("/")[-2]
```

Comment:
@mazeyu This should be replaced with pathlib Path objects, which are multiplatform and would avoid the need for a windows only .patch file